addons/website/views/snippets/s_timeline_images.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template name="Timeline Images" id="s_timeline_images">
+    <section class="s_timeline_images o_colored_level pt48 pb48">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 pb24 text-center" data-name="Heading">
+                    <h2 class="h3-fs">Evolving together</h2>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
+                </div>
+            </div>
+            <div class="position-relative pt-3">
+                <div class="s_timeline_images_row position-relative d-flex flex-column pb-4" data-name="Milestone">
+                    <div class="o_dot_line position-absolute top-0 bottom-0 z-1 w-0 mb-1 border-start pe-none"/>
+                    <span class="o_dot o_not_editable position-absolute translate-middle-x z-1 rounded-circle pe-none text-o-color-1" contenteditable="false"/>
```

Comment:
Do we actually need these `z-1` ? It doesn't seem to change the visual result of the elements

Reply:
It's to place the `o_dot` above the `s_card` in case you change the o_cc color in the `Milestone Event` section of the web editor